inst/stan/linear_single_ind.stan

```diff
@@ -13,10 +13,10 @@ functions{
     real k4;
     real y_hat;
 
-    k1 = DE(y, pars);
-    k2 = DE(y+interval*k1/2.0, pars);
-    k3 = DE(y+interval*k2/2.0, pars);
-    k4 = DE(y+interval*k3, pars);
+    k1 = DE_rk4(y, pars);
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?